src/PrestaShopBundle/Command/ListCommandsAndQueriesCommand.php

```diff
@@ -47,11 +51,31 @@ class ListCommandsAndQueriesCommand extends Command
      */
     private $commandAndQueries;
 
-    public function __construct(CommandDefinitionParser $commandDefinitionParser, array $commandAndQueries)
-    {
+    /**
+     * @var bool
+     */
+    private $isFormatSimple;
```

Comment:
Since a few days, we can add the types on the variables :smiley:  
```suggestion
    private bool $isFormatSimple;
```